### PR TITLE
doc: update acrn_configuration_tool tutorial for RDT feature

### PR DESCRIPTION
**Pull request by liuwei987**
update acrn_configuration_tool tutorial for RDT and update uos type
for launch configuration tutorial.

Signed-off-by: Wei Liu <weix.w.liu@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/tutorials/acrn_configuration_tool.rst [hunk 1]
```diff
@@ -135,6 +135,20 @@ Additional scenario XML elements:
   Specify whether ACRN hypervisor image can be booted using multiboot2 protocol.
   If not set, GRUB's multiboot2 is not available as a boot option.
 
+``RDT_ENABLED`` (a child node of ``FEATURES/RDT``):
+  Specify whether enable Resource Director Technology(RDT) allocation feature.
+  If HW board does not support this feature, UI would have only one 'n' optional.
```
Comment (jsun26intel) [inline doc/tutorials/acrn_configuration_tool.rst:140]: Set to 'y' to enable the feature or 'n' to disable it.
The 'y' will be ignored when hardware does not support RDT.

File: doc/tutorials/acrn_configuration_tool.rst [hunk 2]
```diff
@@ -135,6 +135,20 @@ Additional scenario XML elements:
...
+
+``CDP_ENABLED`` (a child node of ``FEATURES/RDT``):
+  Specify whether enable Code and Data Prioritization(CDP). CDP is an extension
+  of CAT.
+  If HW board does not support RDT or select 'n' for ``RDT_ENABLED``, UI would
+  have only one 'n' optional.
+
```
Comment (jsun26intel) [inline doc/tutorials/acrn_configuration_tool.rst:147]: Set to 'y' to enable the feature or 'n' to disable it.
The 'y' will be ignored when hardware does not support CDP.

File: doc/tutorials/acrn_configuration_tool.rst [hunk 3]
```diff
@@ -135,6 +135,20 @@ Additional scenario XML elements:
...
+``CLOS_MASK`` (a child node of ``FEATURES/RDT``):
+  Specify cache masks value, make sure the filled value is continuous bits.
+  If HW board does not support RDT, then no ``CLOS_MASK`` settings.
+
```
Comment (jsun26intel) [inline doc/tutorials/acrn_configuration_tool.rst:151]: Specify cache capacity bitmask for the CLOS, only continuous '1' bits are allowed.
The value will be ignored when hardware does not support RDT.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/tutorials/acrn_configuration_tool.rst [hunk 4]
```diff
@@ -135,6 +135,21 @@ Additional scenario XML elements:
   Specify whether ACRN hypervisor image can be booted using multiboot2 protocol.
   If not set, GRUB's multiboot2 is not available as a boot option.
 
+``RDT_ENABLED`` (a child node of ``FEATURES/RDT``):
+  Specify whether enable Resource Director Technology(RDT) allocation feature.
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst:139]: change to ``Specify whether to enable ...``

File: doc/tutorials/acrn_configuration_tool.rst [hunk 5]
```diff
@@ -135,6 +135,21 @@ Additional scenario XML elements:
...
+  Set to 'y' to enable the feature or 'n' to disable it.
+  The 'y' will be ignored when hardware does not support RDT.
+
+``CDP_ENABLED`` (a child node of ``FEATURES/RDT``):
+  Specify whether enable Code and Data Prioritization(CDP). CDP is an extension
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst:144]: change to ``Specify whether to enable ...``

**Pull request merged**